styles/components/_masthead.scss

```diff
@@ -19,7 +22,8 @@
 }
 
 /// Adds styles for the Masthead when it is open.
-.sprk-c-Masthead--open {
+.sprk-c-Masthead--open,
+.sprk-c-Masthead--is-open {
```

Comment:
are we using `is-x` as convention now?

Reply:
we have been, it is in the naming convention docs but we didn't follow it in the past.